lib/compress.js

```diff
@@ -3115,11 +3115,19 @@ merge(Compressor.prototype, {
         return self;
     });
 
+    function can_be_extracted_from_if_block(node) {
+        return !(
+            node instanceof AST_Const ||
+            node instanceof AST_Let ||
+            node instanceof AST_Class
+        );
+    }
+
     OPT(AST_BlockStatement, function(self, compressor){
         tighten_body(self.body, compressor);
         switch (self.body.length) {
           case 1:
-            if (!compressor.has_directive("use strict") && compressor.parent() instanceof AST_If
+            if (!compressor.has_directive("use strict") && compressor.parent() instanceof AST_If && can_be_extracted_from_if_block(self.body[0])
```

Comment:
Most of the code fits into 100 columns. Could this be reformatted as follows?
```patch
            if (!compressor.has_directive("use strict")
                && compressor.parent() instanceof AST_If 
                && can_be_extracted_from_if_block(self.body[0]) 
```

Reply:
Done :)